config.rb

```diff
@@ -52,38 +51,31 @@
   # activate :minify_css
 
   # Minify Javascript on build
-  # activate :minify_javascript
-end
-
-# Override generated asset path to remove the long explicit vendor paths
-class ImportedAssetPathProcessor
```

Comment:
we managed to keep this code and just use 

```
//= link_tree ../../node_modules/govuk-frontend/govuk/assets
```

in the Sprockets manifest, wouldn't this work here too?

Reply:
I am a ruby noob, if I did this could I delete my shonky copy stuff I added below, because if so this would be better I guess?

Reply:
I'll try and provide you with a patch/commit, if not let's just roll with what works

Reply:
Et voilà ! https://github.com/alphagov/product-page-example/pull/51 has the changes. It removes the need for a custom Sprockets processor so let me know what you think and no offence if deemed unnecessary 😄